app/helpers/attachments_helper.rb

```diff
@@ -1,4 +1,10 @@
 module AttachmentsHelper
+  include Rails.application.routes.url_helpers
+
+  def default_url_options
+    { host: 'www.gov.uk', protocol: 'https' }
```

Comment:
This should be based on `Plek.website_root` so it will work in all environments: https://github.com/alphagov/plek/blob/master/lib/plek.rb#L105